### PR TITLE
chsrc 0.1.6 (new formula)

### PR DESCRIPTION
**Pull request by Aaron-212**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/chsrc.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  head "https://github.com/RubyMetric/chsrc.git", branch: "main"
+
+  def install
+    if ENV.compiler == :clang
+      system "make", "CC=clang"
+    elsif ENV.compiler == :gcc
+      system "make", "CC=gcc"
+    else
+      system "make"
+    end
```
Comment (chenrui333) [inline Formula/c/chsrc.rb:16]:
```suggestion
    system "make"
```

File: Formula/c/chsrc.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    bin.install "chsrc"
+  end
+
+  test do
+    assert_match("chsrc: Change Source (GPLv3)", shell_output("#{bin}/chsrc --version"))
+    assert_match(/help|list|get|set/, shell_output("#{bin}/chsrc --help"))
```
Comment (chenrui333) [inline Formula/c/chsrc.rb:23]:
```suggestion
```

File: Formula/c/chsrc.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class Chsrc < Formula
+  desc "Change Source for every software on every platform from the command-line"
+  homepage "https://github.com/RubyMetric/chsrc"
+  url "https://github.com/RubyMetric/chsrc/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "212123e79c5b2331dfc994504964d457d1e224e4d4a107a0047a024c1a260028"
+  license "GPL-3.0-only"
```
Comment (chenrui333) [inline Formula/c/chsrc.rb:6]: https://github.com/RubyMetric/chsrc/pull/26

File: Formula/c/chsrc.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  head "https://github.com/RubyMetric/chsrc.git", branch: "main"
+
```
Comment (chenrui333) [inline Formula/c/chsrc.rb:8]: also need livecheck as upstream has rolling latest tag.

Comment (Aaron-212) [reply Formula/c/chsrc.rb:8]: 0.1.6 is marked as latest

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@Aaron-212, thanks for your first contribution to Homebrew-core!! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**